cli/src/main.rs

```diff
@@ -55,9 +58,9 @@ enum Command {
         #[structopt(short, default_value = "1024")]
         memory: u64,
 
-        // Specify multiple disks in groups of 3 targets
-        #[structopt(long, use_delimiter = true)]
-        crucible_target: Vec<SocketAddr>,
+        // file with a JSON array of DiskRequest structs
+        #[structopt(long, parse(from_os_str))]
+        crucible_disks: PathBuf,
```

Comment:
Does this mean that having a crucible disk present is now required?

Reply:
This is my mistake, made it optional in ec97ee3.